cmd/flux/bootstrap_github.go

```diff
@@ -38,11 +38,13 @@ import (
 var bootstrapGitHubCmd = &cobra.Command{
 	Use:   "github",
 	Short: "Bootstrap toolkit components in a GitHub repository",
-	Long: `The bootstrap github command creates the GitHub repository if it doesn't exists and
+	Long: `The bootstrap github command creates the GitHub repository if it doesn't exist and
 commits the toolkit components manifests to the main branch.
 Then it configures the target cluster to synchronize with the repository.
 If the toolkit components are present on the cluster,
-the bootstrap command will perform an upgrade if needed.`,
+the bootstrap command will perform an upgrade if needed.
+Admin permissions are needed on pre-existing repositories
+in order to create a deploy key.`,
```

Comment:
I think the managing of team permissions requires elevated repository permissions as well (as do some other things that happen if you do a strict reconcile, e.g. setting the repository from public to private and vice-versa.

Reply:
I suspect you're right here. I included the deploy key bit as that was the specific one I hit. If it looks like there's going to be a long list happy to remove the specific actions and just cut back to 'Admin permissions are need on pre-existing repositories' : )

Reply:
@hiddeco I've updated it to remove the specific actions and just list the role

Reply:
But that's also not true because of `--token-auth` in combination with not using `--team` will also happily reconcile without administrative permissions I think, as long as you are able to commit :-).

That makes the summary a bit difficult.

Reply:
"Admin permissions may be required on the repository for certain configuration actions to succeed"?

Reply:
I was worried this might be the case. Feels that this may be better suited in a different location, either a specific page in the docs or spread between the specific flags if they modify the defaults?

Reply:
How about "for most configurations admin permissions on the repository are required but this can be lowered by removing requirements to use deploy keys and modify team access"? It's relatively consise and nudges towards the modifications that are being done that might raise/lower permissions (could also link the docs although the URL is huge https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)

Reply:
I think this info is better placed here:  https://github.com/fluxcd/website/blob/main/content/en/docs/installation.md#github-and-github-enterprise

Reply:
> I think this info is better placed here: https://github.com/fluxcd/website/blob/main/content/en/docs/installation.md#github-and-github-enterprise

I'm not sure why I forgot about this page. Thanks!